stdlib/REPL/test/docview.jl

```diff
@@ -46,14 +46,19 @@ end
     checks = ["var", "raw", "r"]
     symbols = "@" .* checks .* "_str"
     results = checks .* "\"\""
-    for (i,r) in zip(symbols,results)
+    for (i, r) in zip(symbols, results)
         @test r ∈ REPL.doc_completions(i)
     end
 end
 @testset "fuzzy score" begin
     # https://github.com/JunoLab/FuzzyCompletions.jl/issues/7
     # shouldn't throw when there is a space in a middle of query
     @test (REPL.matchinds("a ", "a file.txt"); true)
+
+    @test REPL.fuzzyscore("abc", "xyz") == 0
+    @test REPL.fuzzyscore("abc", "abc") == 1
+    @test REPL.fuzzyscore("abc", "dcab") == 0.25
+    @test REPL.fuzzyscore("abc", "dab") ≈ 1 / 3
```

Comment:
These tests seem fragile to me. I would prefer to see
```suggestion
    @test REPL.fuzzyscore("IJilia", "IJulia") >
          REPL.fuzzyscore("IJilia", "MotionCaptureJointCalibration")
    @test REPL.fuzzyscore("bupercalifragilisticexpialidocious", "supercalifragilisticexpialidocious") >
          REPL.fuzzyscore("parasid", "supercalifragilisticexpialidocious")
```
From @jakobnissen's examples in #49466